src/main/java/org/frankframework/frankdoc/model/FrankElement.java

```diff
@@ -138,7 +142,28 @@ private void handlePossibleForwards(FrankClass clazz) {
 				log.warn("FrankElement [{}] has forwards without a name or description", fullName);
 			}
 			this.forwards.add(ParsedJavaDocTag.getInstance(forwardStr));
-		}		
+		}
+	}
+
+	private void handlePossibleTags(FrankClass clazz) {
+		for(String tagStr: clazz.getAllJavaDocTagsOf(JAVADOC_TAG)) {
+			if(StringUtils.isBlank(tagStr)) {
+				log.error("FrankElement [{}] has tags without a name and without a value", fullName);
+			} else if(tagStr.split("[ \\t]").length == 1) {
+				// TODO: This does not work when a quoted tag does not have a value.
+				log.warn("FrankElement [{}] has a tag without a value: [{}]", fullName, tagStr);
```

Comment:
This todo has been expressed in issue #43. Multiple PRs need to be merged before this can be addressed.